gp_fit_real_data/fit_uv_real_data.py

```diff
@@ -84,10 +77,10 @@ def objective_closure():
 
             fixed_noise = np.mean(np.abs(uv_band_flux/snr))
             set_trainable(m.likelihood.variance, False)  # We don't want to optimise the noise level in this case.
-            m.likelihood.variance = fixed_noise
+            m.likelihood.variance.assign(fixed_noise)
```

Comment:
`m.likelihood.variance = fixed_noise` _replaces_ the gpflow.Parameter object with the constant - so then you wouldn't need to do the set_trainable to False anymore. If you want to keep it as a parameter, use `assign()` instead!

Reply:
I see, thanks for the explanation!